### DISCUSSION TITLE
Deleted the user

### DISCUSSION THREAD

**Discussion by Heretolearn-code** [alexjustesen/speedtest-tracker#478] [Q&A]
I deleted the only user thinking it will allow us to load the container without the login wall, but we now cant login as the wall is still ther eand both the default and the previous logins we hafd set dotn work, i also tried restarting the container but still same issue

**Comment by alexjustesen** [comment]
Yep... Auth is required. Guess I need to build a "can't delete the only user" safety mechanism. 

In the interim you'll want to reinstall the application.
